Doc/whatsnew/3.14.rst

```diff
@@ -343,6 +343,12 @@ pathlib
 
   (Contributed by Barney Gale in :gh:`73991`.)
 
+* Add a :attr:`.Path.dir_entry` attribute. In a path object generated by
+  :meth:`.Path.iterdir`, it stores a :class:`os.DirEntry` object corresponding
+  to the path; in other cases it is ``None``.
```

Comment:
```suggestion
* Add the :attr:`.Path.dir_entry` attribute. In a path object generated by
  :meth:`Path.iterdir <pathlib.Path.iterdir>`, it stores an :class:`os.DirEntry`
  object corresponding to the path; in other cases it is ``None``.
```

Reply:
I thinkt it would be more appropriate to change **:attr:`.Path.dir_entry`** to **:attr:`pathlib.Path.dir_entry`**.

Reply:
I don't think the prefix pathlib should be rendered. It's sufficient to have only Path.X and not pathlib.Path.X.